cmd/gateway-gcs.go

```diff
@@ -80,8 +80,13 @@ func gcsMultipartMetaName(uploadID string) string {
 }
 
 // Returns name of the part object.
-func gcsMultipartDataName(uploadID, etag string) string {
-	return fmt.Sprintf("%s/%s/%s", gcsMinioMultipartPathV1, uploadID, etag)
+func gcsMultipartDataName(uploadID string, partNumber int, etag string) string {
+	return fmt.Sprintf("%s/%s/%05d.%s", gcsMinioMultipartPathV1, uploadID, partNumber, etag)
+}
+
+// Returns name of the composed object.
+func gcsMultipartComposeName(uploadID string, composeNumber int) string {
+	return fmt.Sprintf("tmp/%s/%s/composed-object-%05d", gcsMinioMultipartPathV1, uploadID, composeNumber)
```

Comment:
@balamurugana it is like:
for FS   : .minio.sys/tmp/...
for GFS: minio.sys.temp/tmp/...

for GCS we are using minio.sys.temp instead of .minio.sys